bazel/sh_test_wrapper.sh

```diff
@@ -3,4 +3,6 @@
 # Where the runfiles are for tests.
 export TEST_RUNDIR="${TEST_SRCDIR}/${TEST_WORKSPACE}"
 
+cd $(dirname "$0")
```

Comment:
I wanted to pass the output of bazel `select()` to `envoy_sh_test`'s srcs parameter to conditionally disable the the hot restart test. However, `select` only functions when it's passed to a rule. `envoy_sh_test` is a macro and was trying to inspect srcs (something like  `"$(location " + srcs[0] + ")"`), which doesn't work.

I switched those lines to use "$(SRCS)" or to just pass srcs directly, but the different is the absence of the relative path on the front of the filenames.